test/Common.FunctionalTests/Utilities/IISDeploymentResult.cs

```diff
@@ -10,11 +11,14 @@ public class IISDeploymentResult
     {
         public DeploymentResult DeploymentResult { get; }
         public ILogger Logger { get; }
+        public Process HostProcess { get; }
```

Comment:
Would be nice to subclass IISDeploymentResult from DeploymentResult and return IISDeploymentResult from deployers.